shell/common/shell.cc

```diff
@@ -1424,6 +1429,23 @@ bool Shell::OnServiceProtocolGetSkSLs(
   return true;
 }
 
+bool Shell::OnServiceProtocolEstimateRasterCacheMemory(
+    const ServiceProtocol::Handler::ServiceProtocolMap& params,
+    rapidjson::Document* response) {
+  FML_DCHECK(task_runners_.GetRasterTaskRunner()->RunsTasksOnCurrentThread());
+  const auto& raster_cache = rasterizer_->compositor_context()->raster_cache();
+  response->SetObject();
+  response->AddMember("type", "EstimateRasterCacheMemory",
+                      response->GetAllocator());
+  response->AddMember<uint64_t>("layer_bytes",
```

Comment:
nit:  
can you call these `layerBytes` and `pictureBytes` or `layerSizeInBytes` and `pictureSizeInBytes` for consistency with other apis?
using camel case makes the API a little cleaner to use from Dart.

Reply:
Changed to `layerBytes` and `pictureBytes`.